README.md

```diff
@@ -17,12 +17,12 @@ First thing, you need to be an administrator of a Trello Team, then head up for
 you can see the different teams for which you can enable / disable or create new
 power ups. Click on your team name, then click on create a Power-up button.
 
-Give your power up a name and select at least the `card-back-section` capability in
-the list.
+Give your power up a name and select at least the `card-back-section` capability
+in the list.
 
 Finally fill the iframe connector URL field with the adress where your power up
-is hosted. We're using [glitch](https://knp-trello-extension.glitch.me/) to host
-the developpment version of this power-up, but feel free to host it in another
+is hosted. You can use [serveo](https://serveo.net/) to expose your local
```

Comment:
You don't want to speak about Glitch?

Reply:
I will in the article about the project, but it's not necessary in the readme of this project IMHO ;)